src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -204,6 +207,16 @@ void PylonCameraParameter::readFromRosParameterServer(const ros::NodeHandle& nh)
             }
         }
     }
+
+
+    
+    if ( nh.hasParam("whitebalance_auto") )
```

Comment:
Dynamic reconfigure should call your callback with the values set as ROS params at startup - is there a reason you also check the parameter manually here?